pkgs/top-level/all-packages.nix

```diff
@@ -26448,4 +26448,5 @@ in
 
   btcdeb = callPackage ../applications/blockchains/btcdeb {};
 
+  hashi-ui = callPackage ../servers/hashi-ui  {};
```

Comment:
Since this appears to be a server, maybe it would be better served (ha) under the servers header (alphabetically, please): https://github.com/NixOS/nixpkgs/blob/045384ac98ced4f34b735a9dc56c974759d2d52c/pkgs/top-level/all-packages.nix#L15299

All headers start with `###`, so keep it between `### SERVERS` and `### OS-SPECIFIC`: https://github.com/NixOS/nixpkgs/blob/045384ac98ced4f34b735a9dc56c974759d2d52c/pkgs/top-level/all-packages.nix#L6